### ISSUE TITLE
Use react-firebase hooks and refactor the codebase

### ISSUE THREAD

**Issue by abhisheknaiidu** [abhisheknaiidu/class-karona#17]

**Comment by rohan2734** [comment]
Hi i would like to work on this issue. can you assign me?

**Comment by abhisheknaiidu** [comment]
Yea @rohan2734 ,  Go Ahead!

**Comment by rohan2734** [comment]
@abhisheknaiidu can you elaborate the issue? what exactly you want from react-firebase hooks, and  can  you  seperate the task of refractoring the codebase ?I will put a seperate PR on that.

**Comment by abhisheknaiidu** [comment]
@rohan2734  No, you just need to replace all the firebase class component methods and prototypes with the latest react-firebase hooks. 

refer to this documentation for better understanding : https://www.npmjs.com/package/react-firebase-hooks
